packages/zowe-explorer-ftp-extension/src/ZoweExplorerFtpUssApi.ts

```diff
@@ -138,6 +143,10 @@ export class FtpUssApi extends AbstractFtpApi implements ZoweExplorerApi.IUss {
             await Gui.errorMessage("The local directory path provided does not exist.", { logger: ZoweLogger });
             throw new Error();
         }
+
+        // Make directory before copying inner files
+        await this.putContents(inputDirectoryPath, ussDirectoryPath);
```

Comment:
in the ZoweExplorerApi.ts file the `putContents` is deprecated but could add `in favor of putContent` in the JSDoc for awareness and could update the FTP call to putContent here and now. I actually have the work done for it in https://github.com/zowe/vscode-extension-for-zowe/pull/2161 that you can use for code and test fixes when FTP ext changes to IUss.putContent

Reply:
How do we feel about keeping `putContents` and deprecating `putContent`? Like @t1m0thyj has mentioned, it seems inconsistent w/ the naming of `getContents`.

Or, we could deprecate `getContents` and just use the singular form for both.

Reply:
I guess the problem is that the `IUss.putContents` has all of the `IUploadOptions` as parameters instead of a single optional object.
What we could do it keep the inconsistency for V2 (i.e. `IUss.putContents` deprecated in favor of `IUss.putContent`) AND have a message/note in the `IUss.putContent` saying that it will be renamed in V3 😋